ci-operator/step-registry/cucushift/aws/ipi/upgrade/cucushift-aws-ipi-upgrade-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: cucushift-aws-ipi-upgrade
+  steps:
+    pre:
+      - chain: ipi-aws-pre
+      - ref: cucushift-pre
+      - ref: cucushift-upgrade-pre
```

Comment:
Hmm..I still have several questions about the steps here. Maybe we can have a sync later.